sdk/nodejs/appengine/v1/index.ts

```diff
@@ -6,32 +6,28 @@ import * as utilities from "../../utilities";
 
 // Export members:
 export * from "./app";
-export * from "./appAuthorizedCertificate";
-export * from "./appDomainMapping";
-export * from "./appFirewallIngressRule";
-export * from "./appServiceVersion";
+export * from "./authorizedCertificate";
+export * from "./domainMapping";
+export * from "./version";
 
 // Import resources to register:
 import { App } from "./app";
-import { AppAuthorizedCertificate } from "./appAuthorizedCertificate";
-import { AppDomainMapping } from "./appDomainMapping";
-import { AppFirewallIngressRule } from "./appFirewallIngressRule";
-import { AppServiceVersion } from "./appServiceVersion";
+import { AuthorizedCertificate } from "./authorizedCertificate";
+import { DomainMapping } from "./domainMapping";
+import { Version } from "./version";
 
 const _module = {
     version: utilities.getVersion(),
     construct: (name: string, type: string, urn: string): pulumi.Resource => {
         switch (type) {
             case "google-native:appengine/v1:App":
                 return new App(name, <any>undefined, { urn })
-            case "google-native:appengine/v1:AppAuthorizedCertificate":
-                return new AppAuthorizedCertificate(name, <any>undefined, { urn })
-            case "google-native:appengine/v1:AppDomainMapping":
-                return new AppDomainMapping(name, <any>undefined, { urn })
-            case "google-native:appengine/v1:AppFirewallIngressRule":
-                return new AppFirewallIngressRule(name, <any>undefined, { urn })
```

Comment:
Is it expected that this resource was deleted?

I noticed a few other similar removals as well.

Reply:
Great catch! This wasn't intended - I forgot to update `resourceNamePropertyOverrides`. Doing that now...